python-frontend/src/test/java/org/sonar/python/types/TypeShedTest.java

```diff
@@ -488,4 +488,10 @@ public void variables_from_protobuf() throws TextFormat.ParseException {
     assertThat(symbols.values()).extracting(Symbol::kind, Symbol::fullyQualifiedName, Symbol::annotatedTypeName)
       .containsExactlyInAnyOrder(tuple(Kind.OTHER, "mod.foo", "str"), tuple(Kind.OTHER, "mod.bar", null));
   }
+
+  @Test
+  public void stubFilesSymbols_should_not_contain_ambiguous_symbols_of_classes() {
```

Comment:
I'm not sure I understand, I have this test pass even when I revert the implementation of `stubFilesSymbols` to its previous one. Yet, as this change has an actual impact on sonar-security ITs, I wonder how could this test pass without the change?

Reply:
Indeed, my bad. I forgot to populate the cache of typeshed symbols. The tests passes because `Typeshed.typeShedSymbols` is empty in this test